lib/notification.dart

```diff
@@ -15,36 +19,81 @@ class Notification {
   /// A unique identifier because it can be only one alarm.
   static const alarmId = 888;
   final FlutterLocalNotificationsPlugin localNotif =
-      FlutterLocalNotificationsPlugin();
+  FlutterLocalNotificationsPlugin();
 
   /// Add configuration for local notifications and initialize.
   Future<void> init() async {
     const initializationSettingsAndroid =
-        AndroidInitializationSettings('@mipmap/ic_launcher');
-    const initializationSettingsIOS = DarwinInitializationSettings(
-      requestAlertPermission: false,
-      requestBadgePermission: false,
-      requestSoundPermission: false,
-    );
-    const initializationSettings = InitializationSettings(
+    AndroidInitializationSettings('@mipmap/ic_launcher');
+    DarwinInitializationSettings initializationSettingsIOS =
+    DarwinInitializationSettings(
+        requestAlertPermission: false,
+        requestBadgePermission: false,
+        requestSoundPermission: false,
+        notificationCategories: [
+          DarwinNotificationCategory('alarm', actions: [
+            DarwinNotificationAction.text(
+              '0',
+              'snooze',
+              buttonTitle: 'snooze',
+            ),
+            DarwinNotificationAction.text(
+              '1',
+              'stop',
+              buttonTitle: 'stop',
+            ),
+          ])
+        ],
+        onDidReceiveLocalNotification: onDidReceiveLocalNotification);
+    InitializationSettings initializationSettings = InitializationSettings(
       android: initializationSettingsAndroid,
       iOS: initializationSettingsIOS,
     );
 
-    await localNotif.initialize(initializationSettings);
+    await localNotif.initialize(initializationSettings,
+        onDidReceiveNotificationResponse: onDidReceiveNotificationResponse,
+        onDidReceiveBackgroundNotificationResponse: notificationTapBackground);
     tz.initializeTimeZones();
   }
 
+  static void onDidReceiveLocalNotification(
+      int id, String? title, String? body, String? payload) async {
+    // display a dialog with the notification details, tap ok to go to another page
```

Comment:
Please remove all comments. Instead, you should provide a short documentation using `///`, like done with the other methods.